lib/runner/testcase.js

```diff
@@ -160,6 +163,18 @@ TestCase.prototype.setFailed = function(err) {
 };
 
 TestCase.prototype.catchHandler = function(err) {
+  // This reports fewer failures
+  // return this.setFailed(err);
+  // Show exceptions in errors list
+  this.suite.client.errors().push(err.stack);
```

Comment:
This makes it report exceptions in the final report, which we find very handy since we have a lot of tests. We can probably wont turn it into a pr unless there is interest...

* https://github.com/nightwatchjs/nightwatch/pull/1500/files#diff-1739b032bcc4033b3c167ae8765ae39aR125
* https://github.com/nightwatchjs/nightwatch/pull/1500/files#diff-60cebdf36729a4fa8d21cdd8bb6cd817R236